libs/runtime/ebpf_hash_table.c

```diff
@@ -287,8 +320,11 @@ _ebpf_hash_table_compute_bucket_index(_In_ const ebpf_hash_table_t* hash_table,
         length = hash_table->key_size * 8;
         data = key;
     }
-    uint32_t hash_value = _ebpf_murmur3_32(data, length, hash_table->seed);
-    return hash_value & hash_table->bucket_count_mask;
+    if (ebpf_processor_supports_sse42 && (!hash_table->extract)) {
```

Comment:
nit: why do we skip crc if hash table has an `extract` function?

Reply:
Extract can return keys that are not a multiple of 8bits in length, but the CRC instruction assumes 8bit boundaries, hence the check.